tests/queries/0_stateless/02454_ReplacingMergeTreeNewEngine.sql

```diff
@@ -0,0 +1,51 @@
+DROP TABLE IF EXISTS test;
+CREATE TABLE test (uid String, version UInt32, is_deleted UInt8) ENGINE = ReplacingMergeTree(version, is_deleted) Order by (uid);
```

Comment:
Please also add tests with Replicated tables (you can create two replicas in the same server, see other tests)